lib/ansible/config/manager.py

```diff
@@ -122,7 +122,7 @@ def ensure_type(value, value_type, origin=None):
 
         elif value_type == 'pathspec':
             if isinstance(value, string_types):
-                value = value.split(os.pathsep)
+                value = value.split(os.pathsep) if value else ''
```

Comment:
This change doesn't change the behavior at all:
```pycon
$ ipython       
Python 3.9.0 (default, Oct  6 2020, 23:54:30) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import os

In [2]: ''.split(os.pathsep)
Out[2]: ['']
```

Reply:
It becomes an empty list right below this. It could also be 'or []', but that doesn't change the end result.

Reply:
It's usually easier to follow what's going on when vars don't suddenly change their types and stick with one across the scope..